src/data/methods/BIP122.methods.ts

```diff
@@ -2,16 +2,41 @@
  * Methods
  */
 export const BIP122_SIGNING_METHODS = {
-  BIP122_SIGN_MESSAGE: "bip122_signMessage",
```

Comment:
We need to keep this one for backward compatibility on this method we supported for some partners
You can find the original PR here: https://github.com/LedgerHQ/wallet-connect-live-app/pull/174